float64 spatial_dist
float64[] temporal_nov
float64 roi_knowledge
